app/components/op_primer/status_button_component.sass

```diff
@@ -0,0 +1,2 @@
+.op-status-button .ActionListItem-action--leading
```

Comment:
I really don't like that, and I think it is not needed if you remove the `select_variant` from the menu_arguments of the Button (see my other comment).